packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -96,16 +96,20 @@ function BlockBindingsAttribute( { attribute, binding } ) {
 	const { source: sourceName, args } = binding || {};
 	const sourceProps =
 		unlock( blocksPrivateApis ).getBlockBindingsSource( sourceName );
+	const isUndefined = ! sourceProps;
```

Comment:
Nit: this name is unclear, so maybe it should align with the label in the UI:
```suggestion
	const isSourceInvalid = ! sourceProps;
```